packages/components/src/base-control/style.scss

```diff
@@ -12,7 +12,7 @@
 
 	.components-base-control__label {
 		display: inline-block;
-		margin-bottom: $grid-unit-05;
+		margin-bottom: $grid-unit-10;
```

Comment:
This is intentionally a "global" change. As we refine the sidebar overall, 4px is such a small margin it might not even have been there.